src/content/development/security/secrets/_index.en.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Secrets"
+weight: 60
+---
+
+The following secrets are used to store sensitive information
+(with the usual caveat that secrets don't protect sensitive information):
+
+* `broker-secret-` with a Kubernetes-generated suffix, which stores the
+  credentials used to connect to the Broker;
```

Comment:
```suggestion
  credentials used to connect to the Broker
```

Reply:
I think Stephen's way of doing this with a colon before the list and semicolon-split chunks with the normal commas/periods of a sentence is also correct, if not common in our docs.

Reply:
They're bullet points which I don't think typically don't end in any punctuation, ie the preceding bullet provides the logical separation.

Reply:
This is an explanation of the style:

> Lists with points relating to a stem statement
Semicolons were traditionally used in lists to separate each bullet point, but although this is still correct, they are not used as often today. Legal writing is probably the main exception.
If you do use semicolons, the accepted practice, as in this list, is to:
> * put a semicolon at the end of each point;
> * use ‘and’ after the second-to-last point; and
> * finish with a full stop.

https://www.onlinegrammar.com.au/punctuation-in-lists/

I think this might also align with the Chicago Manual of Style.

> The Chicago Manual of Style has pages of rules and examples of bullet lists that agree with the Garner style recommendation to use semicolons after each item, use and after the next-to-last-item, and use a period at the end of the last item. 

https://www.instructionalsolutions.com/blog/bulleted-list-punctuation

I think you're describing something more like AP style, which is what I also typically use.

> AP uses dashes to introduce individual sections of a list, like this:
> - Put a space between the dash or bullet and the first word of each item in the list. 
> - Capitalize the first word following the dash or bullet.
> - Use periods, not semicolons, at the end of each section.

https://twitter.com/apstylebook/status/1415407776726142979?lang=en

Reply:
TBH, I've never used semi-colons in bulleted lists - in fact I wasn't even aware it was a style and don't recall seeing it used anywhere. I looked throughout the Submariner website and didn't see any uses of semi-colons in bulleted lists. Some lists use periods after each item and some don't.

Reply:
Let’s go for periods then.